Video.js

```diff
@@ -190,6 +200,7 @@ export default class Video extends Component {
         type: source.type || '',
         mainVer: source.mainVer || 0,
         patchVer: source.patchVer || 0,
+        requestHeaders: source.headers ? this.stringsOnlyObject(source.headers) : {}
```

Comment:
I believe the key here should be `headers` and not `requestHeaders`. That or `VideoViewManager.PROP_SRC_HEADERS` needs to be changed to `requestHeaders`. Without these changes, this line in VideoViewManager: 

```
src.getMap(PROP_SRC_HEADERS)
```

throws a runtime error due to the fact that 'headers' is not a key on 'src'.